src/styles/index.scss

```diff
@@ -1,3 +1,254 @@
+$header-height: 60px;
+$hover-color: #00acdc;
+$nav-link-color: #060b35;
+
+$buy-button-color: rgb(0, 172, 220);
+$product-att-color: rgb(97, 96, 112);
+
+:root {
+  --column-count: 1;
+}
```

Comment:
Avoid defining custom properties in :root when using SCSS variables. Instead, use SCSS variables for consistency and better maintainability.